app/packages/looker-3d/src/renderables/pointcloud/shaders.tsx

```diff
@@ -131,7 +136,7 @@ const ShadeByIntensityShaders = {
 
     vec4 mvPosition = modelViewMatrix * vec4(pos, 1.0);
 
-    gl_PointSize = pointSize * (1.0 / length(mvPosition.xyz));
+    gl_PointSize = pointSize * (isPointSizeAttenuated ? (1.0 / length(mvPosition.xyz)) : 1.0);
```

Comment:
Not sure I fully understand this - IIUC this means that customizing the point size disables the distance based sizing - or in other words a point near the camera and far from the camera will appear the same size.